### PR TITLE
refs 0000 Artifact Registry setup

### PR DESCRIPTION
**Pull request by osvaldot**

### CODE REVIEW

File: .terraform-docs.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  escape: true
+  hide-empty: false
+  html: true
+  indent: 2
+  lockfile: true
+  read-comments: true
+  required: true
+  sensitive: true
+  type: true
+
```
Comment (Monska85) [inline .terraform-docs.yml]:
```suggestion

```

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+tfsec:
+	docker run --rm -it -v "$$(pwd):/src" aquasec/tfsec /src --tfvars-file=/src/examples/test.tfvars
+
+generate-docs: lint
+	docker run --rm -u $$(id -u) \
+		--volume "$(PWD):/terraform-docs" \
+		-w /terraform-docs \
+		quay.io/terraform-docs/terraform-docs:0.16.0 markdown table --config .terraform-docs.yml --output-file README.md --output-mode inject .
+
```
Comment (Monska85) [inline Makefile]:
```suggestion

```

File: examples/README.md [hunk 3]
```diff
@@ -0,0 +1,2 @@
+# Example
+
```
Comment (Monska85) [inline examples/README.md:2]: Describe the example that you are creating.

File: examples/main.tf [hunk 4]
```diff
@@ -0,0 +1,6 @@
+module "repositories" {
+  source  = "../"
+  version = "~>0.1"
+  project_id = var.project_id
+  repositories = var.repositories
+}
```
Comment (Monska85) [inline examples/main.tf]:
```suggestion
}

```

File: examples/test.tfvars [hunk 5]
```diff
@@ -0,0 +1,25 @@
+
```
Comment (Monska85) [inline examples/test.tfvars]:
```suggestion
```

File: examples/test.tfvars [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+    readers = [
+      "group:dev-team@example.com"
+    ]
+    writers = [
+      "user:admin@example.com"
+    ]
+    location = "europe-west1"
+  },
+  "project-2-images" = {
+    description = "Docker images repository"
```
Comment (Monska85) [inline examples/test.tfvars]:
```suggestion
    description = "Docker images repository 2"
```

File: examples/variables.tf [hunk 7]
```diff
@@ -0,0 +1,16 @@
+variable "project_id" {
+  type        = string
+  description = "The GCP project ID to host the cluster in"
```
Comment (Monska85) [inline examples/variables.tf]:
```suggestion
  description = "The GCP project ID that hosts the Artifact Registry."
```

File: variables.tf [hunk 8]
```diff
@@ -0,0 +1,16 @@
+variable "project_id" {
+  type        = string
+  description = "The project name to deploy the resources."
```
Comment (Monska85) [inline variables.tf]:
```suggestion
  description = "The GCP project ID that hosts the Artifact Registry."
```

**Review by Monska85 — CHANGES_REQUESTED** [review]

File: examples/main.tf (hunk 4 above)
Comment (osvaldot) [reply examples/main.tf]: Thanks

File: examples/README.md (hunk 3 above)
Comment (osvaldot) [reply examples/README.md:2]: Added thanks

**Pull request merged**